charts/tfy-inferentia-operator/templates/neuron-device-plugin.yaml

```diff
@@ -0,0 +1,116 @@
+{{- if .Values.devicePlugin }}
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ .Release.Name }}-neuron-device-plugin
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - nodes
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - ""
+  resources:
+  - events
+  verbs:
+  - create
+  - patch
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  verbs:
+  - update
+  - patch
+  - get
+  - list
+  - watch
+- apiGroups:
+  - ""
+  resources:
+  - nodes/status
+  verbs:
+  - patch
+  - update
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ .Release.Name }}-neuron-device-plugin
+  namespace: {{ .Release.Namespace }}
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ .Release.Name }}-neuron-device-plugin
+  namespace: {{ .Release.Namespace }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: {{ .Release.Name }}-neuron-device-plugin
+subjects:
+- kind: ServiceAccount
+  name: {{ .Release.Name }}-neuron-device-plugin
+  namespace: {{ .Release.Namespace }}
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: {{ .Release.Name }}-neuron-device-plugin
+  namespace: {{ .Release.Namespace }}
+spec:
+  selector:
+    matchLabels:
+      name: {{ .Release.Name }}-neuron-device-plugin-ds
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        name: {{ .Release.Name }}-neuron-device-plugin-ds
+    spec:
+      serviceAccount: neuron-device-plugin
+      tolerations:
+      {{- range .Values.devicePlugin.tolerations }}
+        - {{ . | toYaml | nindent 10 | trim }}
+      {{- else }}
+        []
+      {{- end }}
+      priorityClassName: "system-node-critical"
+      affinity:
+        {{- toYaml (index .Values "devicePlugin" "affinity") | nindent 8 }}
+      containers:
+      - image: .Values.devicePlugin.image
+        imagePullPolicy: Always
+        name: neuron-device-plugin
+        resources:
+          {{- toYaml (index .Values "devicePlugin" "resources") | nindent 10 }}
+        env:
+        - name: KUBECONFIG
+          value: /etc/kubernetes/kubelet.conf
```

Comment:
Weird, is this part of the image itself, don't see it as a mount

Reply:
Removed.